File: ClientApp/src/app/shared/model/Process/Process.ts
import { IProcess } from './IProcess';

export class Process implements IProcess
{
  public id: number;
  public active: boolean;
  public processNumber: string;
  public state: string;
  public monetaryValue: number;
  public startDate : Date;
  public clientId: number;
  public processState: string;
}

File: Models/Process/Process.cs
using System;

namespace Teste_Trinks.Models
{
    public partial class Process : IProcess
    {
        public int Id { get; set; }
        public bool Active { get; set; }
        public string ProcessNumber { get; set; }
        public string State { get; set; }
        public float MonetaryValue { get; set; }
        public DateTime StartDate { get; set;}
        public int ClientId { get; set; }
        public string ProcessState { get; set; }


        public virtual Client Client { get; set; }
    }
}